apps/fluent-tester/src/FluentTester/TestComponents/FocusZone/FocusZoneTest.tsx

```diff
@@ -11,90 +11,68 @@ const directionalFocusZone: React.FunctionComponent<{}> = () => {
   return (
     <View>
       <Button content="Outside FocusZone" />
+      <Text variant="headerSemibold" accessible={false}>FocusZone with Bidirectional arrow key navigation</Text>
```

Comment:
What's the reason for adding `accessible={false}` to these text headers? Does removing it affect the test in any way? Having text be accessible shouldn't give focus to it when assistive technology (like narrator) isn't being used. If it is being used, I think we'd want narrator to focus on and read these headings.

Reply:
I was thinking it would take focus for some reason, brain fart :) Removing now

Reply:
No worries! It's not an uncommon thought. I've seen others use it as such before. Thanks for fixing it!